CHANGELOG.md

```diff
@@ -1,3 +1,302 @@
+# 1.0.10-rc1 (2020-06-23)
+make histogram store data for 1 hour (#3117) (#3119)
+Fix EdgeHub cloud connection (#3112) …
+Fix about edge agent get twin fix (#3111) …
+Enable array support in twin 
+Update SDK to 1.26.0 on release 
+EdgeHub & EdgeAgent would shutdown if Workload API goes stale (#2993) …
+Edge Hub metrics go through asp.net (#3009)
+Fix PriorityQueue e2e tests from dotnet 3.1 update (#2988) …
+Fix host and docker cpu stats (#2879) …
+mitigate versioning error on windows (#3001) …
+Update codebase to dotnet 3.1. (#2996) …
+Use BTreeMap instead of HashMap for config.yaml's hash (#2991) …
+[mqtt] Swap notifier subscriptions order (#2986) …
```

Comment:
I think all mqtt should not be shown in 1.0.10 rc1. Please confirm with Varun.

Reply:
Yes, please remove all the MQTT commits